ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1097,16 +1096,7 @@ void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
                 // unknown coordinate frame
                 break;
             }
-            const Location loc{
-                packet.lat_int,
-                packet.lon_int,
-                int32_t(packet.alt*100),
-                frame,
-            };
-            terrain_alt = (loc.get_alt_frame() == Location::AltFrame::ABOVE_TERRAIN);
-            if (!loc.get_vector_from_origin_NEU(pos_neu_cm)) {
-                break;
-            }
+            loc = Location(packet.lat_int, packet.lon_int, int32_t(packet.alt*100), frame);
```

Comment:
Using braces here rather than prens gives you additional compile-time checks.  That's why they where here originally.

Reply:
ok, thanks for the review! Changed to use parenthesis and re-tested.